.vscode/tasks.json

```
@@ -0,0 +1,80 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Set env to development",
+      "hide": true,
+      "type": "shell",
+      "command": "set",
+      "args": [
+        "${config:jekyll_env.development}"
+      ]
+    },
+    {
+      "label": "Build (Jekyll)",
+      "type": "shell",
+      "command": "bundle",
+      "args": [
+        "exec",
+        "jekyll",
+        "build"
+      ],
+      "problemMatcher": [],
+      "dependsOn": [
+        "Set env to development"
+      ]
```

```suggestion
     "options": {
        "env": {
          "JEKYLL_ENV": "production"
        }
      },
```

Environment variables should be set by `options.env`